tests/test_client_ws.py

```diff
@@ -364,7 +363,7 @@ def read(self, decode=False):
                     await resp.close()
 
 
-async def test_send_data_after_close(ws_key, key_data, loop, mocker) -> None:
+async def test_send_data_after_close(ws_key, key_data, loop) -> None:
```

Comment:
Can we decorate this with `@pytest.mark.parametrize` instead of looping internally? This would be better represented in the test results output then.

Reply:
I don't care too much.
The test did exist, I have no intention to rewrite it.